components/camel-azure/camel-azure-cosmosdb/src/main/docs/azure-cosmosdb-component.adoc

```diff
@@ -2,7 +2,7 @@
 :doctitle: Azure CosmosDB
 :shortname: azure-cosmosdb
 :artifactid: camel-azure-cosmosdb
-:description: To read and write records to the CosmosDB database on the Azure cloud platform.
+:description: To read and write records to the CosmosDB database on Azure cloud platform.
```

Comment:
@orpiske this file changed from the main branch, should I remove it from this backport?

Reply:
No, I think you can rebase to accommodate your changes.